query/lambda/lambda_function_multiple_az_configured.sql

```diff
@@ -1,10 +1,18 @@
+
```

Comment:
@megaproaktiv Can you please remove any empty newlines, like here, and also below at the bottom of the file?

Reply:
sure!